README.md

```diff
@@ -63,107 +63,104 @@ multiple replicas of each of the app deployments.
 
 1. Start by installing the MySQL backend
 
-    ```bash
-    kubectl apply -f k8s/mysql-backend.yml
-    ```
+```bash
+kubectl apply -f k8s/mysql-backend.yml
+```
 
 2. Verify that the mysql-init job successfully completes
 
-    ```bash
-    kubectl get po
-    NAME                    READY     STATUS      RESTARTS   AGE
-    mysql-9bd5bcfdf-7jb2s   1/1       Running     0          3m
-    mysql-init-29nxv        0/1       Completed   0          3m
-    ```
+```bash
+kubectl get po
+NAME                    READY     STATUS      RESTARTS   AGE
+mysql-9bd5bcfdf-7jb2s   1/1       Running     0          3m
+mysql-init-29nxv        0/1       Completed   0          3m
+```
 
 3. Install Linkerd as described above; install the app configured to use MySQL
 
-    ```bash
-    linkerd install | kubectl apply -f -
-    linkerd inject k8s/mysql-app.yml | kubectl apply -f -
-    ```
+```bash
+linkerd install | kubectl apply -f -
+linkerd inject k8s/mysql-app.yml | kubectl apply -f -
+```
 
 4. Use the app!
 
-    ```bash
-    kubectl port-forward svc/webapp 7000
-    open "http://localhost:7000"
-    ```
-
+```bash
+kubectl port-forward svc/webapp 7000
+open "http://localhost:7000"
+```
 
 ### Deploy Chaos Monkey
 
 This repo includes a Chaos Monkey script, that randomly kills a pod every 10
 seconds. It is intended to be run with the Kubernetes / MySQL configuration. To
 deploy, run:
 
-    ```bash
-    kubectl apply -f k8s/mysql-chaos.yml
-    ```
-
----
+```bash
+kubectl apply -f k8s/mysql-chaos.yml
+```
```

Comment:
This looks like the right fix to me. But we only need this fix applied in this section, and the subsequent Service Profiles section, since those are the only two sections not using list elements.